### DISCUSSION TITLE
[bug] "create" button is enabled even without alphanumeric characters

### DISCUSSION THREAD

**Discussion by raynepaws** [community/community#125235] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

if you put a symbol in, then type a letter and delete it during a ~0.1 second window, the "create" button becomes enabled

![screenshot of bug](https://github.com/community/community/assets/129226914/d5744b81-b62f-401a-8424-4ba6c9eefe83)

when clicked, the button becomes disabled and doesn't do anything

**Comment by Khadramaxamuud** [comment]
- [ ]

**Reply by Khadramaxamuud** [reply]
Sawie

**Comment by Ahmadtaha1122** [comment]
Nice good

**Reply by Ahmadtaha1122** [reply]
Good

**Reply by Ahmadtaha1122** [reply]
****

**Comment by Asilkan5807** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> if you put a symbol in, then type a letter and delete it during a ~0.1 second window, the "create" button becomes enabled
> 
> ![screenshot of bug](https://github.com/community/community/assets/129226914/d5744b81-b62f-401a-8424-4ba6c9eefe83)
> 
> when clicked, the button becomes disabled and doesn't do anything

**Comment by Hasannejad** [comment]
https://t.me/hamsteR_kombat_bot/start?startapp=kentId6894963963

**Comment by siruschavoshi** [comment]
> > ### Select Topic Area
> > 
> > Bug
> > 
> > ### Body
> > 
> > if you put a symbol in, then type a letter and delete it during a ~0.1 second window, the "create" button becomes enabled
> > 
> > ![screenshot of bug](https://github.com/community/community/assets/129226914/d5744b81-b62f-401a-8424-4ba6c9eefe83)
> > 
> > when clicked, the button becomes disabled and doesn't do anything
> 
> 
>
